www/src/public/js/controllers/builder/main.js

```diff
@@ -313,11 +319,14 @@
                 baseStats.amulet = encoder.toNumber(listStr.slice(0,1));
             listStr = listStr.substring(1);
 
-            if (statList[0] === '_')
-                baseStats.hazelnut = -1;
-            else
-                baseStats.hazelnut = encoder.toNumber(listStr.slice(0,1));
-            listStr = listStr.substring(1);
+            // v3: Hazelnut
+            if (listVersion == 3) {
+                if (statList[0] === '_')
+                    baseStats.hazelnut = -1;
+                else
+                    baseStats.hazelnut = encoder.toNumber(listStr.slice(0,1));
+                    listStr = listStr.substring(1);
+            }
```

Comment:
We should have an else statement here that sets the hazelnut to spirit if it's not a v3 list, just to maintain the current stats on old lists.

Reply:
I agree.  Completed.